Speaker notes to 6.4 Slices (#163)
google/comprehensive-rust: src/basic-syntax/slices.md

* Slices borrow data from the sliced type.
* Question: What happens if you modify `a[3]`?

<details>

* We create a slice by borrowing `a` and specifying the starting and ending indexes in brackets.

* If the slice starts at index 0, Rust’s range syntax means we can drop the starting index. 
    

> * The same is true for the last index, so `let s: &[i32] = &a[2..len];` or `let s: &[i32] = &a[2..];` are equal.

Reviewer: ```suggestion
* The same is true for the last index, so `&a[2..a.len()]` and `&a[2..]` are equal.
```

> * We set `s` as a reference of `i32`s. Notice that the type of `s` no longer has an array length. This avoids type errors when performing computations on slices of different sizes.

Reviewer: ```suggestion
* `s` is a reference to a slice of `i32`s. Notice that the type of `s` no longer mentions the array length. This allows us to performing computations on slices of different sizes.
```

Author: Thanks Martin, we just need 'perform' vs 'performing'...is it easier if you adjust the change?
